content/en/synthetics/browser_tests/actions.md

```diff
@@ -128,88 +161,98 @@ await promise
 return jQuery().jquery.startsWith('3.5.1')
 ```
 
-**Note**: The way Browser tests load external JavaScript is by adding it to the page, so it will only work if your website accepts it.
+To handle login security questions with custom JavaScript, you can either:
 
-#### Test a downloaded file
+- Manage everything in the JavaScript assertion by reading the security question and inputting the variable
+- Create a [JavaScript variable](#javascript) and a regular browser [`type text` step](#type-text)
 
-Perform verifications on files downloaded in previous steps. You can check that a file was correctly downloaded and assert on: the file name, size, and MD5 value.
+You can also check that the value returned by a JavaScript step is the one you are expecting by running the test or executing JavaScript in the console. 
```

Comment:
```suggestion
You can also check that the value returned by a JavaScript step is what you expect by running the test or executing JavaScript in the console. 
```
(in case you don't end up removing this)